src/sbt-test/universal/multiproject-classifiers/build.sbt

```diff
@@ -0,0 +1,42 @@
+lazy val appVersion = "1.0"
+
+lazy val mySettings: Seq[Setting[_]] =
+  Seq(organization := "org.test", version := appVersion, TaskKey[Unit]("showFiles") := {
+    System.out.synchronized {
+      println("Files in [" + name.value + "]")
+      val files = (target.value / "universal/stage").**(AllPassFilter).get
+      files foreach println
+    }
+  })
+
+lazy val assets = config("assets")
+
+lazy val sub = project
+  .in(file("sub"))
+  .enablePlugins(JavaAppPackaging)
+  .settings(mySettings)
+  .settings(
+    ivyConfigurations += assets,
+    artifact in assets := artifact.value.withClassifier(classifier = Some("assets")),
```

Comment:
The [sbt docs on Aritfacts](http://www.scala-sbt.org/1.0/docs/Artifacts.html) still state there is a `copy` method